examples/operator.rs

```diff
@@ -0,0 +1,100 @@
+extern crate timely;
+extern crate timely_communication;
+
+use std::collections::HashMap;
+
+use timely_communication::Configuration;
+use timely::dataflow::operators::{ToStream, Input, Inspect, Operator, FrontierNotificator};
+use timely::dataflow::channels::pact::Pipeline;
+use timely::progress::timestamp::RootTimestamp;
+use timely::dataflow::Scope;
+
+fn main() {
+    timely::execute(Configuration::Thread, |root| {
+        root.scoped(|scope| {
+            (0u64..10).to_stream(scope)
+                .unary_frontier(Pipeline, "example", |mut builder| {
+                    let mut cap = Some(builder.get_cap(RootTimestamp::new(12)));
+                    move |(input, _frontier), output| {
+                        cap = None;
+                        while let Some((time, data)) = input.next() {
+                            output.session(&time).give_content(data);
+                        }
+                    }
+                }).inspect(|x| println!("{:?}", x));
+        });
+    }).unwrap();
+
+    timely::execute(Configuration::Thread, |root| {
+        root.scoped(|scope| {
+            (0u64..10).to_stream(scope)
+                .unary(Pipeline, "example", |mut builder| {
```

Comment:
Tried this in `https://github.com/frankmcsherry/timely-dataflow/pull/39/commits/b53c69e4074e76c673ef61dcdff03615993dcc23`: as is it requires an additional "use" in files that define operators (to have `InputHandler.next()` and `.foreach()` in scope.